import-export-cli/integration/environmnet_test.go

```diff
@@ -0,0 +1,124 @@
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/base"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+	"testing"
+)
+
+const defaultExportPath = utils.DefaultExportDirName
+
+//List Environments using apictl
+func TestListEnvironments(t *testing.T) {
+	apim := apimClients[0]
+	base.SetupEnvWithoutTokenFlag(t, apim.GetEnvName(), apim.GetApimURL())
+	response, _ := base.Execute(t, "list", "envs")
+	base.GetRowsFromTableResponse(response)
+	base.Log(response)
+	assert.Contains(t, response, apim.GetEnvName(), "TestListEnvironments Failed")
+}
+
+//Change Export directory using apictl and assert the change
+func TestChangeExportDirectory(t *testing.T) {
+	apim := apimClients[0]
+	changedExportDirectory := utils.MockTestExportDirectory + utils.DefaultExportDirName
+	defaultExportPath := utils.DefaultExportDirPath
+
+	args := &setTestArgs{
+		srcAPIM:             apim,
+		exportDirectoryFlag: changedExportDirectory,
+	}
+	output, _ := environmentSetExportDirectory(t, args)
+	base.Log(output)
+
+	//Change value back to default value
+	argsDefault := &setTestArgs{
+		srcAPIM:             apim,
+		exportDirectoryFlag: defaultExportPath,
+	}
+	environmentSetExportDirectory(t, argsDefault)
+	assert.Contains(t, output, "Export Directory is set to", "Export Directory change is not successful")
+}
+
+//Change HTTP request Timeout using apictl and assert the change
+func TestChangeHttpRequestTimout(t *testing.T) {
+	apim := apimClients[0]
+	defaultHttpRequestTimeOut := utils.DefaultHttpRequestTimeout
+	newHttpRequestTimeOut := 20000
+	args := &setTestArgs{
+		srcAPIM:            apim,
+		httpRequestTimeout: newHttpRequestTimeOut,
+	}
+	output, _ := environmentSetHttpRequestTimeout(t, args)
+	base.Log(output)
+
+	//Change value back to default value
+	argsDefault := &setTestArgs{
+		srcAPIM:            apim,
+		httpRequestTimeout: defaultHttpRequestTimeOut,
+	}
+	environmentSetHttpRequestTimeout(t, argsDefault)
+	assert.Contains(t, output, "Http Request Timout is set to", "HTTP Request TimeOut change is not successful")
+}
+
+//Change Token type using apictl and assert the change (for both "jwt" and "oauth" token types)
+func TestChangeTokenType(t *testing.T) {
+	apim := apimClients[0]
+
+	tokenType1 := "oauth"
+	args := &setTestArgs{
+		srcAPIM:       apim,
+		tokenTypeFlag: tokenType1,
+	}
+	output, _ := environmentSetTokenType(t, args)
+	base.Log(output)
+	assert.Contains(t, output, "Token type is set to", "1st attempt of Token Type change is not successful")
+	tokenType2 := "jwt"
+
+	//Change value back to default value with a test
+	argsDefault := &setTestArgs{
+		srcAPIM:       apim,
+		tokenTypeFlag: tokenType2,
+	}
+	output2, _ := environmentSetTokenType(t, argsDefault)
+	base.Log(output2)
+	assert.Contains(t, output2, "Token type is set to", "1st attempt of Token Type change is not successful")
+}
+
+//Login to the environment using email and logout
```

Comment:
In this test case, we expect a user like "abc@gmail.com" to login. So with the tenant domain, the whole username should be like "abc@gmail.com@tenantdomain.com". 
@malinthaprasan WDYT?